RESTSystem/API/CS/CSHandler.py

```diff
@@ -0,0 +1,35 @@
+import types
+import os
+import shutil
+import json
```

Comment:
Yes, right. I need to finish the implementation.
I should have write on the PR "only for discussion" since I did it only to get you know how I was proceeding.